### DISCUSSION TITLE
How to enable Git UI in Obsidian?

### DISCUSSION THREAD

**Discussion by LightTemplar** [Vinzent03/obsidian-git#515] [Problems] [answered]
I have just found this image in the web. Is there UI for Git? I don't have it. How to enable it?
![image](https://github.com/denolehov/obsidian-git/assets/453783/891d5313-987b-4af1-94af-0373ddc93f5b)

**Comment by Vinzent03** [comment, marked as answer]
There is a command to open the source control view as stated in the readme.

**Reply by LightTemplar** [reply]
Thanks!
